### DISCUSSION TITLE
npm start error

### DISCUSSION THREAD

**Discussion by mhassan7599** [npm/rfcs#377] [Question]
> open-react-template@0.1.0 start E:\loudent-main\loudent-main
> react-scripts start
'react-scripts' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! open-react-template@0.1.0 start: `react-scripts start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the open-react-template@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Muhammad Hassan\AppData\Roaming\npm-cache\_logs\2021-05-06T02_56_30_998Z-debug.log

**Comment by SilasRodrigues19** [comment]
Hi @mhassan7599 

I had this same problem with this same package, what I did to solve it was to downgrade the NodeJS version. In my case I uninstalled NodeJS and installed version 14.15.1 LTS.

Here is the installation link, maybe it will solve your problem too

https://nodejs.org/en/blog/release/v14.15.1/

**Comment by Leevandr** [comment]


**Reply by ljharb** [reply]
Please don’t post LLM-generated replies.
